SU-SWS/stanford_profile
# D8 core 1816 wysiwyg tweaks

# READY FOR REVIEW
- 
# Summary
- This is a bunch of spacing tweaks for the WYSIYWG

# Needed By (Date)
- Code freeze

# Urgency
- Not

# Steps to Test

1. Pull this branch
2. Clear cache
3. Verify:

> 1. UL & OR
> margin-bottom: modular spacer 2
> 
> 2. .su-callout-text
> margin-bottom: modular spacer 3
> 
> 3. h2
> padding-top: modular scale 2
> 
> 4. h3, h4
> margin-bottom: modular scale -1
> 
> 5. intro
> margin-bottom: modular scale 2
> 
> 6. th, table td
> line-height: 1.2em;


# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1816
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
